src/test/java/duke/task/DeadlineTaskTest.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class DeadlineTaskTest {
+    @Test
+    public void Test1() {
```

Comment:
Perhaps naming the methods with this format:

whatIsBeingTested_descriptionOfTestInputs_expectedOutcome 
e.g., intDivision_zeroDivisor_exceptionThrown

would better show what is being tested.